### PR TITLE
test: Add symdump file and change install to postinstall in package.json

### PR DESCRIPTION
**Pull request by nalmabrcom**

### CODE REVIEW

**Review by temanbrcom — APPROVED** [review]
🐕

File: tests/package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "dist/**/*.d.ts"
   ],
   "scripts": {
-    "install": "tsc --build tsconfig.json"
+    "postinstall": "tsc --build tsconfig.json"
```
Comment (grianbrcom) [inline tests/package.json:12]: Looks like this affects [installation steps](https://github.com/eclipse/che-che4z-lsp-for-cobol/blob/development/tests/README.md#2-install-all-dependencies).

**Review by grianbrcom — APPROVED** [review]
:tulip:

File: tests/package.json (hunk 1 above)
Comment (nalmabrcom) [reply tests/package.json:12]: Yeah, I'll add in further PR all the info :)

**Pull request merged**